app/models/like.rb

```diff
@@ -1,4 +1,10 @@
 class Like < ApplicationRecord
   belongs_to :author, class_name: 'User'
   belongs_to :post
+
+  after_save :update_likes_counter
+
+  def update_likes_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
- **Suggestion**:
- Here too the same: `Private method`

Reply:
- Well resolved 👍🏽